.changeset/flat-rivers-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/blocks': patch
```

Comment:
Let's change this to `minor`. Since the package's current version is `0.0.1`, changesets will increment this to `0.1.0` and that will be the version that gets published to NPM.